crates/stackable-telemetry/src/tracing.rs

```diff
@@ -48,9 +48,9 @@ pub enum Error {
 /// async fn main() -> Result<(), Error> {
 ///     let _tracing_guard = Tracing::builder()
 ///         .service_name("test")
-///         .with_console_output(LevelFilter::INFO)
-///         .with_otlp_log_exporter(LevelFilter::DEBUG)
-///         .with_otlp_trace_exporter(LevelFilter::TRACE)
+///         .with_console_output("TEST_CONSOLE", LevelFilter::INFO)
+///         .with_otlp_log_exporter("TEST_OTLP_LOG", LevelFilter::DEBUG)
+///         .with_otlp_trace_exporter("TEST_OTLP_LOG", LevelFilter::TRACE)
```

Comment:
suggestion: Adjust the env var for the trace exporter.

```suggestion
///         .with_otlp_trace_exporter("TEST_OTLP_TRACE", LevelFilter::TRACE)
```

Reply:
Fixed in 5b6b1b65e6ad83af5c22f3bafce1a2f5ba78ab22